src/pages/NewChairConsole.js

```diff
@@ -94,32 +97,39 @@ const Records = (props) => {
             const targetTypes = (v.type === 'Chairperson' || v.type === 'DAC');
             return belongsToUser && targetTypes;
           })(electionInfo.votes);
-          return [
-            h(TableTextButton, {
-              key: `vote-button-${e.referenceId}`,
-              onClick: () => props.history.push(`access_review/${dar.referenceId}`),
-              label: 'Vote',
-              disabled: isEmpty(votes)
-            }),
-            h(TableIconButton, {
-              icon: Block,
-              key: `cancel-button-${e.referenceId}`,
-              onClick: () => cancelElectionHandler(electionInfo, dar.referenceId, index)
-            })
-          ];
+          const voteButton = h(TableTextButton, {
+            key: `vote-button-${e.referenceId}`,
+            onClick: () => props.history.push(`access_review/${dar.referenceId}`),
+            label: 'Vote',
+            disabled: isEmpty(votes)
+          });
+          if (chairDacIds.includes(dacId)) {
+            return [
+              voteButton,
+              h(TableIconButton, {
```

Comment:
Instead of wrapping the whole thing in an if, you could set the is rendered property to the result of `chairDacIds.includes(dacId)` and save yourself an if/else block.

Reply:
great idea!!!